### PR TITLE
✨  Add Form Component 

### PR DESCRIPTION
**Pull request by SilviaBarcellini**
The form component appears on the welcome page. The ingredient will be typed inside this field by the user. A minimum length of 4 characters is required to proceed.
- ➕ install React Hook Form
- ✨ add form component
- ✨ add form component stories
- 💄 style form component

### CODE REVIEW

**Review by mewiemers — COMMENTED** [review]
GJ!!🥧

File: components/form/form.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import ReactDOM from "react-dom";
```
Comment (mewiemers) [inline components/form/form.tsx:2]: do you need this?

File: components/form/form.module.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
+.input {
+  width: 100%;
+  padding: 1.5rem;
+  text-align: center;
+  transition: 0.5s;
+  background-size: 200% auto;
+  color: #574848;
```
Comment (phlgr) [inline components/form/form.module.css:7]: This could be a variable

File: components/form/form.tsx (hunk 1 above)
Comment (phlgr) [inline components/form/form.tsx:2]: These aren't needed here

Comment (SilviaBarcellini) [reply components/form/form.tsx:2]: I don't, I'll get rid of it😁

File: components/form/form.module.css (hunk 2 above)
Comment (SilviaBarcellini) [reply components/form/form.module.css:7]: Yes, I'll have to update the css globals, thanks

File: components/form/form.tsx (hunk 1 above)
Comment (SilviaBarcellini) [reply components/form/form.tsx:2]: I'll get rid of it 🔥

**Pull request merged**